Source/Project64/User Interface/Gui Class.cpp

```diff
@@ -1019,24 +983,18 @@ DWORD CALLBACK AboutBoxProc(HWND hWnd, DWORD uMsg, DWORD wParam, DWORD lParam)
 	switch (uMsg) {
 	case WM_INITDIALOG:
 	{
-		enum { ROUND_EDGE = 15 };
-
-		DWORD dwStyle = GetWindowLong(hWnd, GWL_STYLE);
-		dwStyle &= ~(WS_CAPTION | WS_SIZEBOX);
-		SetWindowLong(hWnd, GWL_STYLE, dwStyle);
+		//Title
+		LONG_PTR originalWndProc = GetWindowLongPtrW(hWnd, GWLP_WNDPROC);
+		SetWindowLongPtrW(hWnd, GWLP_WNDPROC, (LONG_PTR)DefWindowProcW);

```

Comment:
oh... That makes so much more sense. Lack of grammar can be so ambiguous :blush: